plugins/inputs/bond/README.md

```diff
@@ -1,6 +1,6 @@
 # Bond Input Plugin
 
-The Bond Input plugin collects bond interface status, bond's slaves interfaces
+The Bond Input plugin collects network bond interface status, bond's slaves interfaces
```

Comment:
this sentence doesn't read as grammatical to me. maybe the oxford comma just before `failures count` would help? the mix of possessive, plural, and third-person present verbs is confusing

Reply:
I only added the word network...the rest is "as-is."

Reply:
https://github.com/influxdata/telegraf/commit/4f1ea13ebf6c0bbe7e79d65a9bb9c56ce0aadcc9